src/process_utility.c

```diff
@@ -1239,6 +1335,15 @@ process_index_start(Node *parsetree)
 		ts_indexing_verify_index(ht->space, stmt);
 	}
 
+	ts_with_clause_filter(stmt->options, HYPERTABLE_NAMESPACE, &hypertable_options, &postgres_options);
+
+	ts_with_clauses_apply(hypertable_options, on_index_arg, index_with_clauses, 1, &hypertable_index_options);
+
+	stmt->options = postgres_options;
+
+	if (hypertable_index_options.multitransaction)
+		PreventTransactionChain(true, "CREATE INDEX ... WITH (hypertable.multi_transaction)");
```

Comment:
This should be PreventInTransactionBlock (see compat.h, was renamed in v11 and we mapped for previous versions as it's more clear anyway)

Reply:
I can never remember if the new version is `Chain` or `Block`